### DISCUSSION TITLE
Upload multiple media files at once

### DISCUSSION THREAD

**Discussion by soeren487** [outline/outline#4951] [Ideas]
It would be nice to be able to upload multiple photos and videos at once. Currently only one media file after another can be added manually on my iPad iOS 16.

**Comment by tommoor** [comment]
Hi, there isn't any restriction in the software on this front – it's quite possible to drag multiple images into a document at once. Is it possible this is a limitation of the iPad?

**Reply by soeren487** [reply]
Hi, thanks a lot for your fast reply! I made some tests today. From Mac and PC it is indeed possible to simply drag & drop several media files at once from Finder/Explorer to the browser window of Outline. But by using the "File Dialog" window it is configured to only choose a single file. Multiple file selection is not configured for this dialog. This seems to be the point why I cannot choose multiple media files at once on the iPad. Can you confirm this?
Thanks in advance!
